pubspec.yaml

```diff
@@ -7,12 +7,17 @@ environment:
   sdk: ^3.1.1
 
 dependencies:
+  code_builder: ^4.7.0
+  dart_style: ^2.3.3
+  geojson: ^1.0.0
+  geopoint: ^1.0.0
```

Comment:
I have an idea about making this package very flexible. You already suggested that the user of the package could be able to define the used geo json file in their `build.yaml` file. I can imagine another option where the user can specify the fields they want to extract from the geojson source.

lets say this is my GEOJSON source:

```json
[
  {
    "type": "Feature",
    "geometry": {
      "type": "Point",
      "coordinates": [125.6, 10.1]
    },
    "properties": {
      "name": "Dinagat Islands"
    }
  }
]
```

Then the user could specify for example the properties (and their type) that should be extracted into a class in their `build.yaml`. It could look like this:

```yaml
source_file: "path/to/my/geojson.json"
extract_properties: {"name": "String"}
```

Based on this you then create your own `GeoFeature` class that contains these properties as fields/variables.
Here it may help again to write a base class that is then extended by `GeoFeature`:

```dart
abstract class GeoFeatureBase {
   final List<Polygon>;
}
```